app/views/constructions/_form.html.erb

```diff
@@ -0,0 +1,134 @@
+              "construction_payload_direction",
+              (@construction.payload || {})["direction"],
+              placeholder: "Norden, Osten, Süden, Westen",
+              class: "form-control",
+              name: "construction[payload][direction]"
+            )%>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Grund</label>
+        <%= text_field_tag(
+              "construction_payload_cause",
+              (@construction.payload || {})["cause"],
+              class: "form-control",
+              name: "construction[payload][cause]"
+            )%>
+      </div>
+    </div>
+  </div>
+
+  <%= render partial: "shared/partials/restrictions_form", locals: { record: construction, record_type: "construction", form: f } %>
+
+  <hr />
+
+  <div class="row">
+    <div class="col">
+      <h2 class="d-sm-flex align-items-center justify-content-between my-4">
+        Kategorien
+      </h2>
+      <p>
+        Jede Baustelle kann in einer oder mehreren Kategorien abgelegt werden.
+      <p>
+
+      <%= render partial: "shared/partials/categories_form", locals: { record: construction, form: f } %>
+
+      <input
+        type="button"
+        value="weitere Kategorie hinzufügen"
+        id="nested-add-category"
+        class="btn btn-sm btn-secondary"
+      />
+    </div>
+  </div>
+
+  <hr />
+
+  <div class="row">
+    <div class="col">
+      <h3 class="d-sm-flex align-items-center justify-content-between my-4">
+        Dateien
+      </h3>
+    </div>
+  </div>
+
+  <%= render partial: "shared/partials/media_contents_form", locals: { record: construction, form: f } %>
+
+  <hr />
+
+  <div class="row">
+    <div class="col-lg-6">
+      <div class="form-group">
+        <label for="description">Latitude</label>
+        <% list_of_locations = construction.locations.presence || [OpenStruct.new(geo_location: nil)] %>
+        <% list_of_locations.each_with_index do |location, index| %>
+          <%= fields_for "construction[locations][#{index}][geo_location]", location.geo_location do |fl|  %>
+            <%= fl.text_field :latitude, class: "form-control" %>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="col-lg-6">
+      <div class="form-group">
+        <label for="description">Longitude</label>
+        <% list_of_locations.each_with_index do |location, index| %>
+          <%= fields_for "construction[locations][#{index}][geo_location]", location.geo_location do |fl|  %>
+            <%= fl.text_field :longitude, class: "form-control" %>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+
+  <hr />
+
+  <%= render(
+        partial: "shared/partials/dates_form",
+        locals: {
+          record: construction,
+          form: f,
+          requireds: ['date_start', 'date_end'],
+          values: {
+            date_start: (Time.now).to_date,
```

Comment:
im `dates_form` partial fehlt noch das setzen eines übergebenen werts für `date_start`

Reply:
done in 3a508fb293c46415156cfcab040cd813cce1214a